README.md

```diff
@@ -1,5 +1,6 @@
-# FSS organisation re-verification process
+This application and it's cloud recources has now been decommissioned
 
+<!--
```

Comment:
Agreed with Ryan.

In this case, commenting out the description won't make sense as this is not the application that has been deployed, and outlived its usefulness, but rather this application is a feature that was completed but never released due to budget cuts and some turmoil.

This README documents all the business context regarding this application that you won't find in any document. So it's best to keep it.